packit/cli/update.py

```diff
@@ -28,9 +28,10 @@
 @click.option(
     "--upstream-git-path",
     help="Path to the upstream git repository.",
-    defult=".")
```

Comment:
I need to name something "defult". Such a lovely name.

I also fixed in #120, one of us will rebase.